### PR TITLE
[DOCS-6259] Release summary and consistency updates

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: desktop-sync/latest/admin/index.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
-title: Administration of Desktop Sync
+title: Administer Desktop Sync
 ---
 
+If you’re an IT administrator, you can configure a number of features to centrally manage your Desktop Sync clients.
```
Comment (oliveralfresco2018) [inline desktop-sync/latest/admin/index.md:5]:
```suggestion
If you’re an I.T. administrator, you can configure a number of features to centrally manage your Desktop Sync clients.
```

Comment (oliveralfresco2018) [reply desktop-sync/latest/admin/index.md:5]: Just trying out the suggestion functionality. Should it be with full stops I dont know?

**Review by oliveralfresco2018 — COMMENTED** [review]
LGTM just minor adjustment possibly

File: desktop-sync/latest/admin/index.md (hunk 1 above)
Comment (anxumalo) [reply desktop-sync/latest/admin/index.md:5]: In general both versions are acceptable, but common usage in our docs is without the full stops.

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

**Pull request merged**